OSDOCS-11330: 4.15.22 rn (#79004)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

// 4.15.22
[id="ocp-4-15-22"]
=== RHSA-2024:4474 - {product-title} 4.15.22 bug fix and security update

Issued: 2024-07-17

> {product-title} release 4.15.22 is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4474[RHSA-2024:4474] advisory. There are no RPM packages in this release.

Author: Yep! Go-live was moved :)

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.15.22 --pullspecs
----

[id="ocp-4-15-22-enhancements"]
==== Enhancements

The following enhancement is included in this z-stream release:

[id="ocp-4-15-22-taskrun-status"]

> ===== Introducing TaskRun status

Author: I'd prefer to keep it for standardization's sake :) but thank you for the feedback!

* Previously, the `TaskRun` status was not displayed near the `TaskRun` name on the *TaskRun details* page. With this update, the `TaskRun` status is located beside the name of the `TaskRun` in the page heading. (link:https://issues.redhat.com/browse/OCPBUGS-32156[*OCPBUGS-32156*])

[id="ocp-4-15-22-bug-fixes"]
==== Bug fixes

* Previously, the `HighOverallControlPlaneCPU` alert triggered warnings based on criteria for multi-node clusters with high availability. As a result, misleading alerts were triggered in {sno} clusters because the configuration did not match the environment criteria. This update refines the alert logic to use {sno}-specific queries and thresholds and account for workload partitioning settings. As a result, CPU utilization alerts in {sno} clusters are accurate and relevant to single-node configurations. (link:https://issues.redhat.com/browse/OCPBUGS-35832[*OCPBUGS-35832*])

> * In an AWS STS cluster, the Cloud Credential Operator (CCO) checks `awsSTSIAMRoleARN` in `CredentialsRequest` to create the secret. Previously, CCO logged an error if `awsSTSIAMRoleARN` was not present, which resulted in multiple errors per second. with this release, CCO does not log the error and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36291[*OCPBUGS-36291*])

Reviewer: ```suggestion
* In an AWS STS cluster, the Cloud Credential Operator (CCO) checks `awsSTSIAMRoleARN` in `CredentialsRequest` to create the secret. Previously, CCO logged an error if `awsSTSIAMRoleARN` was not present, which resulted in multiple errors per second. With this release, CCO does not log the error and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36291[*OCPBUGS-36291*])
```

> * Previously, if a new deployment was performed at the OSTree level on a host that was identical to the current deployment but on a different stateroot, OSTree would treat them as equal. With this release, the OSTree's logic has been modified and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36436[*OCPBUGS-36436*])

Reviewer: ```suggestion
* Previously, if a new deployment was completed at the OSTree level on a host that was identical to the current deployment but on a different stateroot, OSTree treated them as equal. With this release, the OSTree logic is modified and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36436[*OCPBUGS-36436*])
```
- Revised to use simple present tense
- Removed possessive from OSTree

> * Previously, a change of dependency targets introduced in {product-title} 4.14 prevented ARO installs from scaling up new nodes after upgrading to newer versions. With this release, the issue has been resolved for {product-title} 4.15. (link:https://issues.redhat.com/browse/OCPBUGS-36550[*OCPBUGS-36550*])

Reviewer: ```suggestion
* Previously, a change of dependency targets introduced in {product-title} 4.14 prevented {azure-first} {product-title} installations from scaling up new nodes after upgrading to newer versions. With this release, the issue is resolved for {product-title} 4.15. (link:https://issues.redhat.com/browse/OCPBUGS-36550[*OCPBUGS-36550*])
```

> [id="ocp-4-16-2-known-issue_{context}"]

Reviewer 2: ```suggestion
[id="ocp-4-15-22-known-issue_{context}"]
```

==== Known issue

> * If the `ConfigMap` maximum transmission unit (MTU) is absent in the `openshift-network-operator` namespace, you must create the `ConfigMap` manually with the machine MTU value before you start the live migration. Otherwise, the live migration fails. (link:https://issues.redhat.com/browse/OCPBUGS-35829[*OCPBUGS-35829*])

Reviewer 2: ```suggestion
* If the `ConfigMap` object maximum transmission unit (MTU) is absent in the `openshift-network-operator` namespace, you must create the `ConfigMap` object manually with the machine MTU value before you start the live migration. Otherwise, the live migration fails. (link:https://issues.redhat.com/browse/OCPBUGS-35829[*OCPBUGS-35829*])
```

> * If the `ConfigMap` maximum transmission unit (MTU) is absent in the namespace 'openshift-network-operator', users have to create the `ConfigMap` manually with the machine MTU value, before starting the live migration. Otherwise, the live migration will get stuck and fail. (link:https://issues.redhat.com/browse/OCPBUGS-35829[*OCPBUGS-35829*])

Reviewer: ```suggestion
* If the `ConfigMap` maximum transmission unit (MTU) is absent in the `openshift-network-operator` namespace, you must create the `ConfigMap` manually with the machine MTU value before you start the live migration. Otherwise, the live migration fails. (link:https://issues.redhat.com/browse/OCPBUGS-35829[*OCPBUGS-35829*])
```

[id="ocp-4-15-22-updating"]
==== Updating

> To update an existing {product-title} 4.15 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].

Reviewer: ```suggestion
To update an {product-title} 4.15 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```
You can almost always remove the term _existing_ and retain your intended meaning.
